<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findPeopleWindow</class>
 <widget class="QMainWindow" name="findPeopleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>findPeopleWindow</string>
  </property>
  <property name="toolTip">
   <string>视频窗口.</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="videoLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>311</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>文件-&gt;Open File</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="originalButton">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>191</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>原视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recognizeButton">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>识别人脸</string>
    </property>
   </widget>
   <widget class="QPushButton" name="specifiedButton">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>249</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>指定识别</string>
    </property>
   </widget>
   <widget class="QPushButton" name="strangerButton">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>278</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>陌生人</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionStrangers"/>
    <addaction name="actionRegisteredPeoples"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File ...</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>configure ...</string>
   </property>
  </action>
  <action name="actionStrangers">
   <property name="text">
    <string>Strangers ...</string>
   </property>
  </action>
  <action name="actionRegisteredPeoples">
   <property name="text">
    <string>Registered Peoples ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
